pkg/ctl/create/iamidentitymapping.go

```diff
@@ -48,6 +51,7 @@ func createIAMIdentityMappingCmd(cmd *cmdutils.Cmd) {
 		fs.StringSliceVar(&options.Groups, "group", []string{}, "Group within Kubernetes to which IAM role is mapped")
 		fs.StringVar(&options.ServiceName, "service-name", "", "Service name; valid value: emr-containers")
 		fs.StringVar(&options.Namespace, "namespace", "", "Namespace in which to create RBAC resources (only valid with --service-name)")
+		fs.BoolVar(&options.NoShadow, "no-shadow", false, "Turn off shadowing of iam identity mappings mappings")
```

Comment:
I would rather name this some more descriptive. like, `-no-duplicate-mappings` or `-error-on-duplication` or `no-dulpicates`.

Reply:
Updated to `--no-duplicate-arns`